projects/batfish-common-protocol/src/main/java/org/batfish/vendor/VendorConfiguration.java

```diff
@@ -221,26 +220,23 @@ public void undefined(StructureType structureType, String name, StructureUsage u
   }
 
   /* Recursively process children to find all relevant definition lines for the specified context */
-  private static IntStream collectLines(
-      RuleContext ctx, BatfishCombinedParser<?, ?> parser, Map<Integer, Set<Integer>> extraLines) {
-    return IntStream.range(0, ctx.getChildCount())
-        .flatMap(
-            i -> {
-              ParseTree child = ctx.getChild(i);
-              if (child instanceof TerminalNode) {
-                int originalLine = parser.getLine(((TerminalNode) child).getSymbol());
-                return IntStream.concat(
-                    IntStream.of(originalLine),
-                    Optional.ofNullable(extraLines)
-                        .map(el -> el.get(originalLine))
-                        .map(set -> set.stream().mapToInt(Integer::intValue))
-                        .orElse(IntStream.of()));
-              } else if (child instanceof RuleContext) {
-                return collectLines((RuleContext) child, parser, extraLines);
-              }
-              return IntStream.empty();
-            })
-        .distinct();
```

Comment:
Q: how much does this `.distinct()` matter. Having a hard time finding a benchmark that would have invoked it.